src/System.Diagnostics.Process/tests/ProcessStartInfoTests.cs

```diff
@@ -1087,8 +1087,8 @@ public void ShellExecute_Nano_Fails_Start()
                 UseShellExecute = true,
                 FileName = tempFile
             };
-
-            Assert.Throws<PlatformNotSupportedException>(() => Process.Start(info));
+            
+            Assert.Throws<PlatformNotSupportedException>(() => { using (var p = Process.Start(info)) { } });
```

Comment:
Using isn't needed here; you'll never get to the Dispose because Start throws. (And if it doesn't throw, then the test will fail.